Detectors/EMCAL/simulation/src/Patches.cxx

```diff
@@ -0,0 +1,306 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file Patches.cxx
+/// \brief Implementation of the EMCAL Patches for the LZEROElectronics
+#include <cstring>
+#include <gsl/span>
+#include <fairlogger/Logger.h> // for LOG
+#include "EMCALSimulation/Patches.h"
+#include "EMCALBase/TriggerMappingV2.h"
+
+using namespace o2::emcal;
+
+//____________________________________________________________________________
+Patches::Patches() : mPatchSize(2), mWhichSide(0), mWhichSuperModuleSize(0)
```

Comment:
I would rename the class to `TRUElectronics`, because that is essentially what you want to simulate.

Reply:
Done